docs/drivers/drivers.md

```diff
@@ -0,0 +1,59 @@
+## Drivers Maintenance 
+
+#### How to upgrade the Nvidia drivers in your cluster
+
+
+###### Available GPU drivers version in OpenPAI
+
+```
+"384.111"  (Default Value)
+```
+
+
+###### Cluster Configuration
+
+- Choose an available version in Openpai, and change the ```drivers``` section in ```service-configuration.yaml```.  
+
+- Update the configuration in your cluster with the command ```paictl config push```. If you wanna investigate more detail about this command, please refer to this [link](../paictl/paictl-manual.md#Config_Push) 
+
+###### Stop corresponding service
+
+- Delete hadoop-node-manager
+    - Note: all running jobs will be killed and retry after upgrading
+       
+```
+./paictl service delete -n hadoop-node-manager
+```
+
+   
+- Stop monitoring service
```

Comment:
[optional suggestion] seem later paictl support paictl stop -n [service1, service2, service3 ..], batch start and stop services. Could be useful for admin.

Reply:
Sure